metrics/examples/prometheus/install/strimzi-service-monitor.yaml

```diff
@@ -16,7 +16,7 @@ spec:
   # configured in additional secret
 
   #### job_name: kube-state-metrics
-  - port: metrics
+  - port: prometheus
```

Comment:
Why this change? @stanlyDoge @ppatierno is this ok with you?

Reply:
The port name was changed in an earlier PR: https://github.com/strimzi/strimzi-kafka-operator/blob/dc05e17698e940d5b51844bfd6ee0f72c73b0a93/cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

Reply:
Is it the right link? I do not see any related change there.